frontend/src/components/debiai/dataAnalysis/widgets/PointPlot/PointPlot.vue

```diff
@@ -1144,13 +1198,22 @@ export default {
       this.sizeAxisSelection = false;
       this.pointPlotDrawn = false;
     },
-    swap() {
+    async swap() {
       let memoryLinePlotDrawn = this.linePlotDrawn;
+      // Swap the columns
       let temp = this.columnYindex;
       this.columnYindex = this.columnXindex;
       this.columnXindex = temp;
+      // Ask the user if he accepts the risk then swap the columns
```

Comment:
`// Ask the user if he accepts the risk then swap the columns`
Replace by
`// Ask the user if he accepts the risk`